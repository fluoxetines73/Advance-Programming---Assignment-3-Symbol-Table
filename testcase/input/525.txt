BEGIN
INSERT f_FF number
BEGIN
END
RPRINT
LOOKUP f_FF
ASSIGN f_FF '1234ABC'
END
RPRINT
INSERT a number