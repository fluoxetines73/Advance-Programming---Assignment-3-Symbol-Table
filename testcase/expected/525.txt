TypeMismatch: ASSIGN f_FF '1234ABC'